### DISCUSSION TITLE
PHP 8.0 Support?

### DISCUSSION THREAD

**Discussion by ashtonlclark** [add2cal/add-to-calendar-button#331] [Q&A] [answered]
Hello, I recently updated my website from PHP 7.4 to PHP 8.0. As soon as I did that, the add to calendar button seemed to stop working. Can you confirm if the code works with PHP 8.0? If not, is there something I can do to get it working in PHP 8.0?

@jekuer .

Thank you.

**Comment by jekuer** [comment, marked as answer]
The Add to Calendar Button has no dependency to your PHP version. So, there needs to be another reason for the issue.

How are you using it? With the WordPress plugin (also running with PHP8 on my side) or via the CDN?

**Reply by ashtonlclark** [reply]
We are using Wordpress and grabbing the latest Add to Calendar code via the CDN.

**Reply by jekuer** [reply]
Ok. This then should definitely make no difference. Maybe a plugin is blocking something?
You could try to install our official plugin instead. It also speeds up your site by keeping the code locally. Maybe this fixes your issue by accident.

http://add-to-calendar-button.com/en/use-with-wordpress

**Reply by ashtonlclark** [reply]
Thanks we’ll try that. I’ll have to find the plugin.

**Reply by jekuer** [reply]
in doubt, search for "add2cal"

**Comment by ashtonlclark** [comment]
Thank you. We were able to fix it without the plugin. Thanks for your help.

**Reply by jekuer** [reply]
perfect. Just to be sure: was it somehow related to the Add to Calendar Button or only a side effect caused by something else?

Btw: would still recommend the plugin for WP ;-)

**Reply by ashtonlclark** [reply]
This issue was due to some PHP 8.0 related issues on our end. Your code is great. I’ll check it out. It may be harder for us to use it because of our custom integration of the add to calendar button.

**Reply by jekuer** [reply]
Thanks.
It should still work. You can use the plugin style or your existing one. Since the only base action is loading the script, it should work both ways.

**Reply by ashtonlclark** [reply]
Thanks, okay, we will take a look at that as well.
